plugins/generator-1.17.1/forge-1.17.1/jsontriggers/item_damaged.json.ftl

```diff
@@ -3,7 +3,9 @@
   "conditions": {
     "items": [
       {
-        "item": "${input$item}",
+        "items": [

```

Comment:
But here it is twice items items, on other places it is item, items, is this ok, was this tested?

Reply:
As I said, this is what I tested and it worked.

Reply:
I have tested this condition and advancement was granted by default. Check https://minecraft.fandom.com/wiki/Advancement/JSON_format#minecraft:item_durability_changed, but this also affects others that use items instead of item on top level

Reply:
Just saw it is an item damaged. I thought we were still on the placed block problem. Sorry for this. It may have a problem.